xudo/commands/prune.py

```diff
@@ -0,0 +1,13 @@
+"""The prune command."""
+
+
+from .base import Base
+
+import os
+
+
+class Prune(Base):
+    """Prompts the user to prune their docker environment"""
+
+    def run(self):
+        os.system("docker system prune")
```

Comment:
This is a nice cmd if you want to go one step further beyond just cleaning up dangling images